test_configs/sockets/quick.test

```diff
@@ -106,15 +106,17 @@
 {
 	prov_name: sockets,
 	test_type: [
+		FT_TEST_LATENCY,
 		FT_TEST_BANDWIDTH,
 	],
 	class_function: [
 		FT_FUNC_WRITE,
 		FT_FUNC_WRITEV,
-		FT_FUNC_WRITEMSG,
```

Comment:
I figured I would leave it out since there's a problem with the readmsg and writemsg